frontend/src/app/App.scss

```
@@ -74,4 +74,8 @@ body,
   &.pf-m-drilldown:not(.pf-m-scrollable) {
     overflow: hidden;
   }
+  // Make tables inside menus transparent for consistent background
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)